batch/snippets/src/test/java/com/example/batch/CreateResourcesIT.java

```diff
@@ -251,6 +258,67 @@ public void createBatchCustomEventTest()
                     .anyMatch(runnable -> runnable.getDisplayName().equals(displayName))));
   }
 
+  @Test
```

Comment:
I think some unrelated changes during merge conflict resolution was applied to this PR. Or did you mean to include the tests?